File: src/CovidStatistics.java
public class CovidStatistics {
    private int activeCases;
    private int confirmed;
    private int deceased;
    private int recovered;

    public CovidStatistics(int activeCases, int confirmed, int deceased, int recovered) {
        this.activeCases = activeCases;
        this.confirmed = confirmed;
        this.deceased = deceased;
        this.recovered = recovered;
    }

    public int getActiveCases() {
        return activeCases;
    }

    public int getConfirmed() {
        return confirmed;
    }

    public int getDeceased() {
        return deceased;
    }

    public int getRecovered() {
        return recovered;
    }

}

File: Covid19 Analysis/Covid19 Analysis/CovidStatistics.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Covid19_Analysis
{
    class CovidStatistics
    {
        private int activeCases { get; set; }
        private int confirmed { get; set; }
        private int deceased { get; set; }
        private int recovered { get; set; }

        public CovidStatistics(int activeCases, int confirmed, int deceased, int recovered)
        {
            this.activeCases = activeCases;
            this.confirmed = confirmed;
            this.deceased = deceased;
            this.recovered = recovered;
        }
    }
}
